src/Stratis.Sidechains.Networks/CirrusV2/CirrusMain.cs

```diff
@@ -36,7 +36,7 @@ internal CirrusMain()
             this.DefaultConfigFilename = NetworkDefaultConfigFilename;
             this.MaxTimeOffsetSeconds = 25 * 60;
 
-            var consensusFactory = new SmartContractPoAConsensusFactory();
```

Comment:
I see all the Cirrus networks now use `SmartContractCollateralPoAConsensusFactory`. What uses `SmartContractPoAConsensusFactory` then? (Just trying to understand the hierarchy)

Reply:
>What uses SmartContractPoAConsensusFactory then?

It can be used for pure PoA + SC